### DISCUSSION TITLE
Final Group Assignment

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#39] [Assignments]
### Due 6/8 (or 6/1*)

Please use this thread to submit your [final group assignment](https://albertkun.github.io/22S-ASIAAM-191A/assignments/final/).

## Submission

- [Select a presentation slot on the final sheet](https://docs.google.com/spreadsheets/d/1amjsdypbGuijuiQtI44UNupEzJET94IFLNp0FVMT01w/edit?usp=sharing)
- Separately take the [individual group evaluation](https://forms.office.com/r/j46zZywbgV)
- Post your group's repository and website link below:

ex.
Group Repo: https://github.com/akochaphum/my-awesome-group
Group website: https://akochaphum.github.io/my-awesome-group/index.html

**Comment by spolcharoen** [comment]
Group Repo: https://github.com/penelopelamb/wasted-opportunities
Group website: https://penelopelamb.github.io/wasted-opportunities/index.html

**Reply by penelopelamb** [reply]
😻

**Comment by ntran27** [comment]
Repo: https://github.com/ntran27/22s-asiaam-191a-group
Website: https://ntran27.github.io/22s-asiaam-191a-group/final/index.html

**Comment by shubhgupta2018** [comment]
Repo: https://github.com/shubhgupta2018/uclgay
Website: https://shubhgupta2018.github.io/uclgay/index.html

**Comment by kericsp** [comment]
Repo: https://github.com/emmashanson/asianam191AUC-residentsin-LA
Website: https://emmashanson.github.io/asianam191AUC-residentsin-LA/final/index.html

**Comment by antonyblin** [comment]
Repo: https://github.com/kaittli/aanhpi-healthcare
Website: https://kaittli.github.io/aanhpi-healthcare/index.html

**Comment by bittertang** [comment]
Repo: https://github.com/bittertang/UCLAsianStories
Website: https://bittertang.github.io/UCLAsianStories/index.html

**Comment by codingdianat** [comment]
repo: https://github.com/stefuntran/AAPI-Discrimination
website: https://stefuntran.github.io/AAPI-Discrimination/

**Comment by joonyloony** [comment]
repo: https://github.com/joonyloony/TAXEVADERS
website: https://joonyloony.github.io/TAXEVADERS/index.html

**Comment by angelineeow** [comment]
repo: https://github.com/lindseymardona/group-8
website: https://lindseymardona.github.io/group-8/final/index.html
